examples/3d_examples/OctopusArmCase3D/run_tapered_arm_and_sphere_with_flow.py

```diff
@@ -194,25 +194,25 @@ def tapered_arm_and_cylinder_flow_coupling(
         )
         io_dx = flow_sim.dx * np.ones(grid_dim)
         io_grid_size = np.array(grid_size)
+        # Initialize flow eulerian grid IO
         io = IO(dim=grid_dim, real_dtype=real_t)
         io.define_eulerian_grid(origin=io_origin, dx=io_dx, grid_size=io_grid_size)
         io.add_as_eulerian_fields_for_io(
             vorticity=flow_sim.vorticity_field, velocity=flow_sim.velocity_field
         )
-        # Initialize sphere IO
-        rod_io = IO(dim=grid_dim, real_dtype=real_t)
+        # Initialize forcing grid IO
+        forcing_grid_io = IO(dim=grid_dim, real_dtype=real_t)
         # Add vector field on lagrangian grid
-        rod_io.add_as_lagrangian_fields_for_io(
+        forcing_grid_io.add_as_lagrangian_fields_for_io(
             lagrangian_grid=cosserat_rod_flow_interactor.forcing_grid.position_field,
             lagrangian_grid_name="rod_forcing_grid",
             vector_3d=cosserat_rod_flow_interactor.lag_grid_forcing_field,
         )
```

Comment:
this can be removed.

Reply:
Yeah that can be removed; I left it there in case one wants to do analysis on the forcing grid itself. But okay, I'll go ahead and remove it.

Reply:
Fixed in [722f451](https://github.com/SophT-Team/SophT-Simulator/pull/70/commits/722f451c2f4e0b8a43da93ede8af488d35a7eba5)